### PR TITLE
fix: use non-default contract query costs

### LINKED ISSUE #12481: (mod) Use expected query costs for `getContractBytecode`/`getContractInfo`
### Description

These queries are using a default cost which greatly exceeds their canonical costs.

### Steps to reproduce

Run these queries with a non-privileged payer.

### Additional context

_No response_

### Hedera network

previewnet, other

### Version

`develop`

### Operating system

None

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Closes #12481

### CODE REVIEW

**Review by povolev15 — APPROVED** [review]
LGTM. thank you @tinker-michaelj

**Review by iwsimon — APPROVED** [review]
LGTM. Thanks @tinker-michaelj

**Review by lukelee-sl — APPROVED** [review]
LGTM

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractGetBytecodeSuite.java [hunk 1]
```diff
@@ -71,21 +78,47 @@ public boolean canRunConcurrent() {
...
+                .then(
+                        withOpContext((spec, opLog) -> {
+                            final var getBytecode = getContractBytecode(contract)
+                                    .payingWith(CIVILIAN_PAYER)
+                                    .saveResultTo("contractByteCode")
+                                    .exposingBytecodeTo(bytes -> {
+                                        canonicalQueryFeeAtActiveRate.set(
+                                                spec.ratesProvider().toTbWithActiveRates((long)
+                                                        (canonicalUsdFee * 100 * TINY_PARTS_PER_WHOLE)));
+                                        log.info(
```
Comment (netopyr) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractGetBytecodeSuite.java:98]: Some leftover?

**Review by lukelee-sl — APPROVED** [review]
LGTM

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractGetBytecodeSuite.java (hunk 1 above)
Comment (tinker-michaelj) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractGetBytecodeSuite.java:98]: Yes probably not helpful, will delete in next PR.

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**